tests/test_neuron_functional.py

```diff
@@ -293,12 +289,12 @@ def test_breaker_of_tmd_algo():
 
     assert_array_equal(N.apical_sections, [33])
     assert_array_almost_equal(
-        n.sections[169].points[-1],
+        n.sections[118].points[-1],
```

Comment:
Why indices changed here?

Reply:
Because of this: https://github.com/BlueBrain/NeuroTS/pull/47#discussion_r959399932